pkg/config/generate.go

```diff
@@ -35,68 +79,128 @@ func selectNodeImage(cs *acsapi.ContainerService, c *Config) {
-		c.TemplateServiceBrokerImage = "docker.io/openshift/origin-template-service-broker:v3.10"
-		c.PrometheusNodeExporterImage = "openshift/prometheus-node-exporter:v0.15.2"
-		c.RegistryImage = "openshift/origin-docker-registry:v3.10.0"
-		c.RouterImage = "openshift/origin-haproxy-router:v3.10.0"
 		c.AzureCLIImage = "docker.io/microsoft/azure-cli:latest"
-		c.RegistryConsoleImage = "cockpit/kubernetes:latest"
-		c.AnsibleServiceBrokerImage = "ansibleplaybookbundle/origin-ansible-service-broker:latest"
-		c.WebConsoleImage = "openshift/origin-web-console:v3.10.0"
-		c.OAuthProxyImage = "openshift/oauth-proxy:v1.0.0"
-		c.PrometheusImage = "openshift/prometheus:v2.2.1"
-		c.PrometheusAlertBufferImage = "openshift/prometheus-alert-buffer:v0.0.2"
-		c.PrometheusAlertManagerImage = "openshift/prometheus-alertmanager:v0.14.0"
 	}
 }
 
 func selectContainerImagesOSA(cs *acsapi.ContainerService, c *Config) {
+	// TODO:
+	// Publish tunnel, sync, images
+	// After GA change ImageConfigFormat, default registry and update default tags for released version
+	c.ImageConfigFormat = "openshift3/ose-${component}:${version}"
 	switch cs.Properties.OrchestratorProfile.OpenShiftConfig.OpenShiftVersion {
-	//TODO: confirm minor version after release
 	case "3.10":
-		c.MasterEtcdImage = "rhel7/etcd:v3.10.15-1"
-		c.MasterAPIImage = "openshift3/ose-control-plane:v3.10.15-1"
-		c.MasterControllersImage = "openshift3/ose-control-plane:v3.10.15-1"
-		c.NodeImage = "openshift3/ose-node:v3.10.15-1"
-		c.ServiceCatalogImage = "openshift3/ose-service-catalog:v3.10.15-1"
-		c.TemplateServiceBrokerImage = "openshift3/ose-template-service-broker:v3.10.15-1"
-		c.PrometheusNodeExporterImage = "openshift3/prometheus-node-exporter:v3.10.15-1"
-		c.RegistryImage = "openshift3/ose-docker-registry:v3.10.15-1"
-		c.RouterImage = "openshift3/ose-haproxy-router:v3.10.15-1"
-		c.RegistryConsoleImage = "openshift3/registry-console:v3.10.15-1"
-		c.AnsibleServiceBrokerImage = "openshift3/ose-ansible-service-broker:v3.10.15-1"
-		c.WebConsoleImage = "openshift3/ose-web-console:v3.10.15-1"
-		c.OAuthProxyImage = "openshift3/oauth-proxy:v3.10.15-1"
-		c.PrometheusImage = "openshift3/prometheus:v3.10.15-1"
-		c.PrometheusAlertBufferImage = "openshift3/prometheus-alert-buffer:v3.10.15-1"
-		c.PrometheusAlertManagerImage = "openshift3/prometheus-alertmanager:v3.10.15-1"
-		c.TunnelImage = "docker.io/jimminter/tunnel:latest"      //TODO: We need to publish it somewhere.
-		c.SyncImage = "docker.io/jimminter/sync:latest"          //TODO: We need to publish it somewhere.
-		c.AzureCLIImage = "docker.io/microsoft/azure-cli:2.0.41" //TODO: create mapping for OSA release to any other image we use
+		c.ImageConfigFormat = "registry.reg-aws.openshift.com/openshift3/ose-${component}:${version}"
+		c.ContainerPullSecret = configureDevRegPullSecret("registry.reg-aws.openshift.com")
+
+		c.MasterEtcdImage = "registry.reg-aws.openshift.com/rhel3/etcd:3.2.22"
```

Comment:
For sanity's sake/auditability I think it's going to make sense to somehow pull out strings like `registry.reg-aws.openshift.com/openshift3` here to avoid repetition

Reply:
Yes, eventaually we will need this for PR testing too